testing/tests/DevExpress.ui.widgets.editors/checkbox.tests.js

```diff
@@ -160,6 +161,74 @@ QUnit.module('Checkbox', function() {
             assert.equal($element.find(ICON_SELECTOR).height(), initHeight, 'icon height is not resized');
         });
 
+        if(isRenovation) {
+            QUnit.test('checkbox icon must resize according to the "iconWidth" and "iconHeight" options', function(assert) {
+                const $element = $('#checkBox').dxCheckBox();
+                const instance = $element.dxCheckBox('instance');
+                const newSize = 50;
+
+                instance.option('iconWidth', newSize);
+
+                assert.strictEqual($element.find(ICON_SELECTOR).outerWidth(), newSize, 'icon width is resized');
+
+                instance.option('iconHeight', newSize);
+
+                assert.strictEqual($element.find(ICON_SELECTOR).outerHeight(), newSize, 'icon height is resized');
+            });
+
+            QUnit.test('checkbox icon\'s font-size must change according to the "iconWidth" and "iconHeight" options', function(assert) {
+                const $element = $('#checkBox').dxCheckBox();
+                const instance = $element.dxCheckBox('instance');
+
+                const newSize = 50;
+                const initialFontSize = $element.find(ICON_SELECTOR).css('font-size');
+
+                instance.option('iconWidth', newSize);
+                instance.option('iconHeight', newSize);
+
+                const newFontSize = $element.find(ICON_SELECTOR).css('font-size');
+
+                assert.notStrictEqual(initialFontSize, newFontSize, 'font-size is changed');
```

Comment:
Cannot we check fontSize calculation algorithm here? The fact that initial and new font size are not equal is not very useful

Reply:
done